docs/CONTRIBUTORS.rst

```diff
@@ -1,92 +1,22 @@
-
-The required ``Signed-off-by`` text can be automatically appended to the commit
-message via the ``-s`` command-line option to ``git commit``:
-
-::
-
-  $ git commit -s -m "Commit message"
-
-The full text of the DCO:
-
-::
-
-    Developer Certificate of Origin
-    Version 1.1
-
-    Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
-    1 Letterman Drive
-    Suite D4700
-    San Francisco, CA, 94129
-
-    Everyone is permitted to copy and distribute verbatim copies of this
-    license document, but changing it is not allowed.
-
-    Developer's Certificate of Origin 1.1
-
-    By making a contribution to this project, I certify that:
-
-    (a) The contribution was created in whole or in part by me and I have the
-    right to submit it under the open source license indicated in the file; or
-
-    (b) The contribution is based upon previous work that, to the best of my
-    knowledge, is covered under an appropriate open source license and I have
-    the right under that license to submit that work with modifications,
-    whether created in whole or in part by me, under the same open source
-    license (unless I am permitted to submit under a different license), as
-    indicated in the file; or
-
-    (c) The contribution was provided directly to me by some other person who
-    certified (a), (b) or (c) and I have not modified it.
-
-    (d) I understand and agree that this project and the contribution are
-    public and that a record of the contribution (including all personal
-    information I submit with it, including my sign-off) is maintained
-    indefinitely and may be redistributed consistent with this project or the
-    open source license(s) involved.
-
-
-To facilitate development and installation of edited version of the code base,
-developers are encouraged to install `Virtualenv <https://virtualenv.pypa.io/en/latest/index.html>`_,
-which is a tool to create isolated Python environments.  It includes
-``pip`` and ``setuptools``, Python packages that can be used to
-install TUF and its dependencies. All installation methods of
-virtualenv are outlined in the `installation
-section <https://virtualenv.pypa.io/en/latest/installation.html>`_,
-and instructions for installing locally from source are provided here:
-::
-
-    $ curl -O https://pypi.python.org/packages/source/v/virtualenv/virtualenv-15.0.3.tar.gz
-    $ tar xvfz virtualenv-15.0.3.tar.gz
-    $ cd virtualenv-15.0.3
-    $ python3 virtualenv.py myVE
-
+Contribute to python-tuf by submitting pull requests against the "develop"
+branch of this repository. Detailed instructions are available in our
+`development guidelines
+<https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md>`_.
+All submitted code should follow our `style guidelines
+<https://github.com/secure-systems-lab/code-style-guidelines/blob/master/python.md>`_
+and must be unit tested [reference to test section below] .
```

Comment:
> [reference to test section below]
Hehe, I actually meant to make "unit tested" a link to the test section below. I didn't know the rst syntax form the top of my head. :D

Reply:
Oh, I'm sorry! Actually, it was a regression after I tried to migrate it to Markdown.